[2025-05-10 13:44:22,257][saicinpainting.utils][WARNING] - Setting signal 1 handler <function print_traceback_handler at 0x3109c7740>
[2025-05-10 13:44:22,257][__main__][CRITICAL] - Prediction failed due to [Errno 2] No such file or directory: '/Users/sangeetsatpathy/Desktop/Experiment/lama/LaMa_models/lama-celeba-hq/lama-regular/config.yaml':
Traceback (most recent call last):
  File "/Users/sangeetsatpathy/Desktop/Experiment/lama/bin/predict.py", line 47, in main
    with open(train_config_path, 'r') as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sangeetsatpathy/Desktop/Experiment/lama/LaMa_models/lama-celeba-hq/lama-regular/config.yaml'

